pkg/util/mount/nsenter_mount.go

```diff
@@ -189,9 +189,10 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, nil
 	}
 	strOut := strings.TrimSuffix(string(out), "\n")
-
 	glog.V(5).Infof("IsLikelyNotMountPoint findmnt output for path %s: %v", file, strOut)
-	if strOut == file {
+
+	//TODO: This is a temporary workaround for fixing the buggy code. Track the solution with issue #26996
+	if strings.Contains(file, strOut) {
```

Comment:
if file is /a/likely/mountpoint-111 and strOut is /a/likely/mountpoint-11,
then strings.Contains(file, strOut) returns true and it is a mount point.

On Thu, Jun 9, 2016 at 6:49 PM, Paul Morie <notifications@github.com> wrote:

> In pkg/util/mount/nsenter_mount.go
> <https://github.com/kubernetes/kubernetes/pull/27054#discussion_r66551038>
> :
>
> >  	glog.V(5).Infof("IsLikelyNotMountPoint findmnt output for path %s: %v", file, strOut)
> > -	if strOut == file {
> > +
> > +	//TODO: This is a temporary workaround for fixing the buggy code. Track the solution with issue #26996
> > +	if strings.Contains(file, strOut) {
>
> I'm not sure this is going to fix things.
>
> What happens if file is /a/likely/mountpoint-111 and strOut is
> /a/likely/mountpoint-11?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27054/files/201576f0aa8467f75308880148973bc6618704b5#r66551038>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASSNxUGwbDoBrLa53g4lfUT_BGQ-s3Boks5qKMLBgaJpZM4IxMdY>
> .
>



-- 
- Jing